.yarnrc.yml

```diff
@@ -4,3 +4,16 @@ enableTelemetry: false
 httpTimeout: 60000
 nodeLinker: node-modules
 npmRegistryServer: 'https://registry.yarnpkg.com'
+
+# logFilters codes described: https://yarnpkg.com/advanced/error-codes
+logFilters:
```

Comment:
Questions:

- isn't there a way to properly fix or handle these warning, instead of ignoring them?
- likely these also happen in the JupyterLab / Notebook repos? If so should we suggest updating the configuration there? So the JupyterLite repo does not do things too differently
- let's move this yarn config (and the discussion) to a separate PR, so it will be easier to track and find it in the future?

Reply:
> properly fix or handle these warning

Most (by count in an install) aren't warnings, but rather low-level play-by-play of needing to/downloading/filling the cache. Indeed, it is almost certainly making installs _slower_ even trying to formulate the entries, even if they don't get printed

The warnigns that _do_ remain, and fit on a page (as posted) are _potentially_ worth investigating, which can only be determined by _actually_ showing the log, _reading_ what is says, and recognizing the shape.

> happen in the JupyterLab / Notebook repos?

It looks like upstream might relegate the `yarn` call to inside `hatch` (inside another, temporary environment), none of which are cached, so that logs only make it out if it full explodes, and maybe not then.

This happens so often, is so slow, and is so noisy, i reckon folk just had to tune it out. But it makes trying to find anything in build logs rather awful. i'm assuming nobody looks unless it fully breaks the build, in which case one is debugging hatch, special non-standard hatch stuff, yarn, lerna, node, etc. all at the same time.

This is also all the more reason to restore the loss of `YARN_HISTORY` being `node_modules/.yarn-state.yml`, so that install is called less frequently.

Reply:
> This is also all the more reason to restore the loss of `YARN_HISTORY` being `node_modules/.yarn-state.yml`, so that install is called less frequently.

Maybe also worth suggesting upstream?

Reply:
Pushed a small commit to align the comment with the one in https://github.com/jupyterlab/jupyterlab/pull/15214.